### DISCUSSION TITLE
tiktok.com

### DISCUSSION THREAD

**Discussion by Daxy7** [uBlockOrigin/uAssets#19048] [General] [answered]
What does this div placeholder do on TikTok?

```
https://www.tiktok.com/@illegalbugtrader?lang=en
```

![1u1vlvs](https://github.com/uBlockOrigin/uAssets/assets/91396482/7ad5a4b0-43b0-45c9-b7e8-5de9389f2863)

**Comment by Sonic5435325** [comment, marked as answer]
Seems like it's just there to space out the "Log in" section and the footer. Going on inspect element shows there's nothing else associated with the placeholder.
![Screenshot from 2023-07-17 00-38-58](https://github.com/uBlockOrigin/uAssets/assets/89158249/15d0c7c9-74c4-42ae-870f-868cf79d6746)
